### PR TITLE
Fix #650: make README and CONTRIBUTING documentations up-to-date again

### LINKED ISSUE #650: README is outdated

### PR DESCRIPTION
**Pull request by scottinet**
Fix #650 

* README: update introduction, install instructions, URLs and tests documentation. Add a fork section to document how to install default plugins from git
* CONTRIBUTING: updarte documentation links and development instructions. Also anticipated the HTTP/WS/SocketIO port merging in the documentation.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,41 +3,40 @@
...
+docker-compose -f docker-compose/dev.yml up
 ```
 
-You can now access to:
-
-* http://localhost:7511 for the standard Kuzzle HTTP API
-* ws://localhost:7512 for the Socket.io API
-* ws://localhost:7513 for the raw websocket API
-* http://127.0.0.1:8080/debug?port=7000 to debug the server
+You can now access to `http://localhost` for the standard Kuzzle HTTP, WebSocket and Socket.io APIs
```
Comment (ballinette) [inline CONTRIBUTING.md]: shouldn't we keep a custom port by default (the same for both protocols, I agree) ?
I mean, I'm not sure that we can expose the port 80 on the host if we are not root.

Comment (scottinet) [inline CONTRIBUTING.md]: Good point. What port did you choose?

Comment (ballinette) [inline CONTRIBUTING.md]: we could keep 7511, for example ?

Comment (scottinet) [inline CONTRIBUTING.md]: Ok, I'll update the CONTRIBUTING file.

**Pull request merged**